setup.py

```diff
@@ -109,7 +113,7 @@ def find_package_data(
 AUTHOR = "Colin Powell"
 AUTHOR_EMAIL = "colin@onecardinal.com"
 URL = "http://github.com/powellc/django-notes/"
-VERSION = __import__(PACKAGE).__version__
+VERSION = __import__(NAME).__version__
```

Comment:
I can't comment on lines you didn't change...

On [line 129](https://github.com/sohonetlabs/django-notes/pull/3/files#diff-2eeaed663bd0d25b7e608891384b7298R129) should `PACKAGE` also be changed to `NAME`?
I guess the real question is, where is `setup.py` run from?